### ISSUE TITLE
Global upload size limit per hour 

### ISSUE THREAD

**Issue by RX14** [maxpowa/npomf#14]
There should be a limit for the whole site of upload data per hour, preferably rolling limit. This prevents the drive filling up. This limit being hit should send an email (or Pushbullet) to the owner.

Might also be a good idea to stop uploads completely when it hits a certain amount of disk space left.

**Comment by clickbait** [comment]
i think it should maybe replace all of the text on the homepage with marquees and blinking text for nostalgia purposes

**Comment by johncave** [comment]
@RX14 
https://www.nagios.org
http://www.zabbix.com
Zabbix can even send you a message on XMPP when something's triggered. Future is now.

**Comment by RX14** [comment]
Yes but npomf would still need to trigger nagios or zabbix. Besides both suites are pretty heavy for single-server setups imo.

**Comment by johncave** [comment]
I used to run Zabbix to monitor a 512mB digital ocean machine (customer wanted it) so I'm sure yours can handle it. Set it to trigger if ingress is above a threashold for the hour or disk usage of the uploads directory exceeds a given amount and problem solved. 

Would be nice to limit the size the uploads directory can grow to, though.
